src/lib/openapi/spec/project-environment-schema.ts

```diff
@@ -3,14 +3,21 @@ import { FromSchema } from 'json-schema-to-ts';
 export const projectEnvironmentSchema = {
     $id: '#/components/schemas/projectEnvironmentSchema',
     type: 'object',
-    additionalProperties: false,
```

Comment:
I could only find this type being used as a post body, but I'll have an extra look

Reply:
Well, it was. it's not anymore. I'll re-add this